spec/unit/rom/memory/relation_spec.rb

```diff
@@ -44,19 +50,34 @@
     end
 
     it 'restricts data using block' do
-      expect(relation.restrict { |tuple| tuple[:age] > 10 }).to match_array([
+      expect(relation.restrict { |tuple| tuple[:age].to_i > 10 }).to match_array([
         { name: 'Jade', email: 'jade@doe.org', age: 11 },
-        { name: 'Joe', email: 'joe@doe.org', age: 12 }
+        { name: 'Joe',  email: 'joe@doe.org',  age: 12 },
+        { name: 'Jack',                        age: 11 }
       ])
     end
   end
 
   describe '#order' do
     it 'sorts data using provided attribute names' do
-      expect(relation.order(:name).to_a).to eq([
-        { name: 'Jade', email: 'jade@doe.org', age: 11 },
-        { name: 'Jane', email: 'jane@doe.org', age: 10 },
-        { name: 'Joe', email: 'joe@doe.org', age: 12 }
+      expect(relation.order(:age, :email).to_a).to eq([
+        { name: 'Jane', age: 10, email: 'jane@doe.org' },
+        { name: 'Jade', age: 11, email: 'jade@doe.org' },
+        { name: 'Jack', age: 11                        },
+        { name: 'Joe',  age: 12, email: 'joe@doe.org'  },
+        { name: 'Jill',          email: 'jill@doe.org' },
+        { name: 'John'                                 }
+      ])
+    end
+
+    it 'places nil before other values when required' do
+      expect(relation.order(:age, :email, nils_first: true).to_a).to eq([
+        { name: 'John'                                 },
+        { name: 'Jill',          email: 'jill@doe.org' },
+        { name: 'Jane', age: 10, email: 'jane@doe.org' },
+        { name: 'Jack', age: 11                        },
+        { name: 'Jade', age: 11, email: 'jade@doe.org' },
+        { name: 'Joe',  age: 12, email: 'joe@doe.org'  }
```

Comment:
I'd like to see a non-nil secondary value sort in these example, to make sure that that's getting ordered as well.

So, maybe another:  `{ name: 'Jaques', age: 11, email: 'jaques' }` thrown into the mix, to make sure that the secondary key sorts, beyond just partitioning on nil